src/d3d9/d3d9_surface.cpp

```diff
@@ -88,6 +88,11 @@ namespace dxvk {
       return S_OK;
     }
 
+    if (riid == __uuidof(ID3D9VkInteropBuffer)) {
+      *ppvObject = ref(m_texture->GetVkBufferInterop());
+      return S_OK;
+    }
```

Comment:
Is it necessary to expose implementation details like this? Not that it's likely that we ever move away from textures having buffers, but I don't really want this to end up making any changes at all more difficult.

Reply:
Yeah, I suppose I can't think of a use for this specific hook at the moment.